tests/GoogleAuthenticatorTest.php

```diff
@@ -39,10 +39,17 @@ public function testGetCode()
         $this->assertTrue($this->helper->checkCode('3DHTQX4GCRKHGS55CJ', $code));
     }
 
-    public function testGetUrl()
+    public function testGetUrlIssuer()
     {
         $url = $this->helper->getUrl('foo', 'foobar.org', '3DHTQX4GCRKHGS55CJ', 'FooBar');
         $expected = 'https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=otpauth://totp/foo@foobar.org%3Fsecret%3D3DHTQX4GCRKHGS55CJ%26issuer%3DFooBar';
         $this->assertEquals($expected, $url);
     }
+
+    public function testGetUrlNoIssuer()
+    {
+        $url = $this->helper->getUrl('foo', 'foobar.org', '3DHTQX4GCRKHGS55CJ');
+        $expected = 'https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=otpauth://totp/foo@foobar.org%3Fsecret%3D3DHTQX4GCRKHGS55CJ';
+        $this->assertEquals($expected, $url);
```

Comment:
I've copied the style from a function that was already there... You are making this the hardest PR I've ever done... :S